gemrb/core/Strings/StringView.h

```diff
@@ -50,7 +50,7 @@ class StringViewImp {
 
 	// explicit because strlen is inefficient
 	explicit StringViewImp(CharT* cstr) noexcept
-	: StringViewImp(cstr, std::strlen(cstr)) {}
+	: StringViewImp(cstr, std::char_traits<CharT>::length(cstr)) {}
```

Comment:
could you just add something like `Traits = std::char_traits<CharT>` and use it where appropriate?

Reply:
Ok, yes